.gitignore

```diff
@@ -1,5 +1,4 @@
-### https://raw.github.com/github/gitignore/f2ce448f2ba7a092da05482ceca99209127c0884/maven.gitignore
-
+# maven
```

Comment:
Nah...I'll work around it...plus it's a change to an SVN project, so I've got to copy in your changes. That reminds me, maybe I'll take collections and promote it to being a git repo in the apache infrastructure.